### ISSUE TITLE
Future Wars (1989) Manual

### ISSUE THREAD

**Issue by SmilingSpectre** [exoscoriae/eXoDOS#4678]
We haven't proper manual. What we have is installation guide only and need to be replaced, I think.

Unfortunately, I wasn't able to find PC manual - but found three slightly different Amiga/Atari manuals. I think, they will do. Select the needed ones yourself.

Future Wars (1989) (Amiga).pdf. Best graphically. Features extra manual protection: you must cover the screen with transparent mesh and select the right square. Not applicable to PC version, I believe, and can be overlooked.

Future Wars (1989) (Amiga) (Kixx Release).pdf. Same as previous, but each pictures has needed mesh embedded already. We have it already as "Alternate manual". Maybe worth to be replaced with one above?

Future Wars (1989) (Amiga-Atari).pdf. "Real" manual. Features also the picture we has as "copy-protection.pdf", only bigger and better.

**Comment by exoscoriae** [comment]
I'm not really a fan of any of these. 

also, the copy protection from amiga does not carry over to the dos version, if I remember correctly. They changes some of the colors or numbers based on talk I saw on the scummvm forums years ago.

we have the atari-amiga version as an alternate manual already in the extras

**Comment by SmilingSpectre** [comment]
@exoscoriae 
As usual, I wasn't able to convey my thoughts, when there is more than one. Sorry. I'll try to rephrase.

1. We have "Alternate manual.pdf", and it's good, but Future Wars (1989) (Amiga).pdf I uploaded before is exactly the same by content, but haven't picture divided by squares (we don't need it anyway). I also prefer its quality - it's slightly smaller, but existing one has moire, and, well, the grid. I again recommend to replace our Alternative Manual, but check it yourself. I reuploaded it.

2. Amiga features totally different protection (mentioned "grid"). But manual I mentioned has exactly the same picture as we has as "copy-protection". We already has the manual, so I simply clarifying here.
